masterlist.yaml

```diff
@@ -1040,8 +1040,29 @@ plugins:
     tag:
       - NoMerge
   - name: 'More Perks.esm'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/35979' ]
     tag:
       - Names
+  - name: 'More Perks for Dead Money.esm'
+    dirty:
+      - <<: *quickClean
+        crc: 0x129204A2
+        util: '[FNVEdit v4.0.1i EXPERIMENTAL](https://www.nexusmods.com/newvegas/mods/34703)'
+        itm: 301
+    clean:
```

Comment:
A plugin can't be clean and also have wild edits... The wild edit list items should also come under `dirty:`.

Reply:
👍 I'll update that

Reply:
Should be fixed. If there are any other complaints please let me know.,

Reply:
I don't see any changes, did you commit them to the right branch / push the branch?

Reply:
https://github.com/gutfuckllc/falloutnv/commit/021fba321ccbf79238c1fbfba04509423283360e

It appears on my end. Not sure what's wrong here. Git is such an headache sometimes

Reply:
Should I close and reopen a pull request?